hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ContractHTSSuite.java

```diff
@@ -89,7 +110,12 @@ public List<HapiSpec> getSpecsInSuite() {
     }
 
     List<HapiSpec> negativeSpecs() {
-        return List.of(nonZeroTransfersFail());
+        return List.of(
+                nonZeroTransfersFail(),
+                transferTokensNegativeCasesFail(),
+                transferTokenNegativeCasesFail(),
+                transferNFTsNegativeCasesFail(),
+                transferNFTNegativeCasesFail());
     }
 
     List<HapiSpec> positiveSpecs() {
```

Comment:
How can it be that there are _no_ positive tests?  Not a single happy path?  Is it covered in some other suite? (Not suggesting that _this_ PR change for that, just wondering if we need an issue for it.)

Reply:
Hey!
Seems like we test the `transferNFT, transferNFTs, transferToken and transferTokens` as positive cases in:

`ContractKeysStillWorkAsExpectedSuite.approvalFallbacksRequiredWithoutTopLevelSigAccess`

Reply:
We need an index or something to the suites and the tests.  It's hard to decide where to _put_ a test, and it's hard to decide where to _find_ a test, unless you know all the suites ....